docs/software/media.md

```diff
@@ -1648,4 +1649,87 @@ The Snapcast client will prompt you for the server's IP and port when installing
 
 Source code: <https://github.com/badaix/snapcast>
 
+## Navidrome
+
+An open source web-based music collection server and streamer, written in Go.  
+It is API compatible to the [Subsonic media player](http://www.subsonic.org/pages/index.jsp).
+
+![Navidrome web interface screenshot](../assets/images/dietpi-software-media-navidrome.png){: width="400" height="202" loading="lazy"}
+
+=== "Web interface"
+
+    The web interface uses port **4533**:
+
+    - URL: `http://<your.IP>:4533`
+
+=== "Multiple Music Folders"
+
+    Navidrome don't support multiple music folders out of the box. The feature is still being developed.
```

Comment:
```suggestion
    Navidrome doesn't support multiple music folders out of the box. The feature is still being developed.
```

Reply:
Take over this change in the Releasenotes PR.